vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -91,9 +96,25 @@ func main() {
 	}
 
 	healthCheck := metrics.NewHealthCheck(time.Minute)
-	metrics.Initialize(*address, healthCheck)
 	metrics_admission.Register()
 
+	go func() {
+		pathRecorderMux := mux.NewPathRecorderMux("vertical-pod-autoscaler-admission-controller")
+
+		if *enableProfiling {
+			routes.Profiling{}.Install(pathRecorderMux)
```

Comment:
Is depending on the apiserver package the best practice here? I'm not sure if that's entirely safe as they may add profiling things that don't fully match.
Might there be something similar focused on api clients (as we're not really apiservers, even though at least the admission controller is serving).

Reply:
Yeah, valid point.
I can't any client side library that includes a pprof function that can be reused. 
Since your other comment is suggesting that this chunk of code gets moved to a central place to be re-used, I'll just manually setup the pprof paths there, rather than bringing in a new dependancy

Reply:
Done in 7ec04b330830836ed483572edc9c1dad3c087f38

Something I realised, is that I copied the cluster-autoscaler way of adding pprof, see: https://github.com/kubernetes/autoscaler/blob/77bf5809ce159f1e88d5576b26e5de51cc94a557/cluster-autoscaler/main.go#L689-L691

I do agree that for adding pprof it doesn't make sense to add an entire new dependency, so may be that's an opportunity to go clean up cluster-autoscaler too.